contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -716,7 +723,15 @@ drop_bbf_db(const char *dbname, bool missing_ok, bool force_drop)
 		{
 			Node	   *stmt = ((RawStmt *) lfirst(parsetree_item))->stmt;
 			PlannedStmt *wrapper;
+			is_set_userid = false;
 
+			if(stmt->type != T_DropRoleStmt && stmt->type != T_GrantStmt)
```

Comment:
why cant we simply check if stmt->type is dropdb cmd?

Reply:
So the stmts for which we want to set current_user = dbo are
"DROP SCHEMA dummy CASCADE"
"DROP OWNED BY dummy, .. CASCADE"